website/routes.py

```diff
@@ -337,6 +339,14 @@ def make_url_map(app):
             json_renderer,
         )
     ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/confirmed_emails/', 'get', auth_views.verified_email_get, json_renderer),
```

Comment:
As suggested in https://github.com/CenterForOpenScience/osf.io/pull/5528/files/2cb0c6854f964a4381be36744e6744300b5ea7d6#r61778717 , just make it a helper function or a property. There's no need for a view.